stan/math/fwd/core/fvar.hpp

```diff
@@ -39,46 +39,45 @@ namespace math {
 template <typename T>
 struct fvar {
   /**
-   * The value of this variable.
-   */
-  T val_;
-
-  /**
-   * The tangent (derivative) of this variable.
+   * The type of values and tangents.
    */
-  T d_;
+  using Scalar = std::decay_t<T>;
```

Comment:
```suggestion
  using ScalarT = std::decay_t<T>;
```

Bit more consistent with the rest of the library

Reply:
Just doing a grep for `ScalarT` vs. `Scalar` I think we use `Scalar` more often? I was trying to keep it lined up with Eigen.

Reply:
Ah that's right. I was thinking more about the naming conventions we use for template types, safe to ignore